### PR TITLE
Enable strict types throughout

### PR DESCRIPTION
**Pull request by gsteel**
Also fixes a couple of bugs surfaced by turning strict types on.

It's worth reviewing commits in isolation due to the number of changed files…

### CODE REVIEW

File: src/File/Size.php [hunk 1]
```diff
@@ -251,7 +254,7 @@ public function isValid($value, $file = null)
 
         // limited to 4GB files
         ErrorHandler::start();
-        $size = sprintf('%u', filesize($fileInfo['file']));
+        $size = (int) sprintf('%u', filesize($fileInfo['file']));
```
Comment (froschdesign) [inline src/File/Size.php]: Looks like a duplication: type casting and `sprintf` with `%u`?!

Comment (gsteel) [reply src/File/Size.php]: Yes, but psalm does not recognise the coercion:

https://psalm.dev/r/0fd48ff1ba

Comment (froschdesign) [reply src/File/Size.php]: And killing `sprintf` is not a solution here?

Comment (gsteel) [reply src/File/Size.php]: Good point! The error handler can get in the bin too 😂

**Review by froschdesign — APPROVED** [review]

**Pull request merged**